<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpgradesTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>690</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Pakcet</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>111</y>
      <width>431</width>
      <height>631</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>33</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string>BitGradeApp</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>drivers</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>bitup</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>info</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>bit</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>fla</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>ini</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>txt</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>pres</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>cond</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>wav</string>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>pm</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>so</string>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="12" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>sh</string>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="13" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>msg</string>
     </property>
    </item>
    <item row="14" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="14" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="15" column="0">
     <property name="text">
      <string>bmp</string>
     </property>
    </item>
    <item row="15" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="15" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="16" column="0">
     <property name="text">
      <string>BitPlatFormApp</string>
     </property>
    </item>
    <item row="16" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="16" column="2">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="17" column="0">
     <property name="text">
      <string>HomeApp</string>
     </property>
    </item>
    <item row="17" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="17" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="18" column="0">
     <property name="text">
      <string>SysCtrl</string>
     </property>
    </item>
    <item row="18" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="18" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="19" column="0">
     <property name="text">
      <string>SaCtrlApp</string>
     </property>
    </item>
    <item row="19" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="19" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="20" column="0">
     <property name="text">
      <string>RtsaCtrlApp</string>
     </property>
    </item>
    <item row="20" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="20" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="21" column="0">
     <property name="text">
      <string>LTE_FDD_CTL</string>
     </property>
    </item>
    <item row="21" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="21" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="22" column="0">
     <property name="text">
      <string>LTE_TDD_CTL</string>
     </property>
    </item>
    <item row="22" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="22" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="23" column="0">
     <property name="text">
      <string>CalcNotic</string>
     </property>
    </item>
    <item row="23" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="23" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="24" column="0">
     <property name="text">
      <string>SaCalcAnaApp</string>
     </property>
    </item>
    <item row="24" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="24" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="25" column="0">
     <property name="text">
      <string>routemap</string>
     </property>
    </item>
    <item row="25" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="25" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="26" column="0">
     <property name="text">
      <string>NR_CTL</string>
     </property>
    </item>
    <item row="26" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="26" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="27" column="0">
     <property name="text">
      <string>RtsaCalcAnaApp</string>
     </property>
    </item>
    <item row="27" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="27" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="28" column="0">
     <property name="text">
      <string>google</string>
     </property>
    </item>
    <item row="28" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="28" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="29" column="0">
     <property name="text">
      <string>dtb</string>
     </property>
    </item>
    <item row="29" column="1">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="29" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="30" column="0">
     <property name="text">
      <string>uImage</string>
     </property>
    </item>
    <item row="30" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="30" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="31" column="0">
     <property name="text">
      <string>BOOT.bin</string>
     </property>
    </item>
    <item row="31" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="31" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="32" column="0">
     <property name="text">
      <string>rtl8188eufw.bin</string>
     </property>
    </item>
    <item row="32" column="1">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="32" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 154, 6);</string>
    </property>
    <property name="text">
     <string>File Folder:0
File:1
Extension:2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Property</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>67</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 154, 6);</string>
    </property>
    <property name="text">
     <string>Yes:1
No:0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>634</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>20191204</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>640</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>594</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>V0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>600</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sversion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>556</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hversion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>550</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A0.1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>420</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string> Sversion/Hversion
 should be managed 
 through database</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>712</x>
      <y>90</y>
      <width>41</width>
      <height>641</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>311</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Configure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Property:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Update:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
